python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -305,8 +312,8 @@ def process_node_data(process_confs, arr_merger, remap_id,
         (feat_ops, two_phase_feat_ops, after_merge_feat_ops, _) = \
             parse_feat_ops(process_conf['features'], process_conf['format']['name']) \
                 if 'features' in process_conf else (None, [], {}, [])
-        label_ops = parse_label_ops(process_conf, is_node=True) \
-                if 'labels' in process_conf else None
+        label_ops, label_mask_fields = parse_label_ops(process_conf, is_node=True) \
+                if 'labels' in process_conf else (None, None)
```

Comment:
And another thing is that maybe we can directly refer the label_mask_fields in the label ops? I think mask_fields are passed into the label_ops, so that we could avoid too many changes later.

Reply:
For the first question: this is a good point. We can add some check when parsing the config files.

Reply:
For the second question, we can get the marks by iterating over label_ops. But I think the implementation complexity is similar.

Reply:
Updated